src/main/java/io/cryostat/net/reports/ArchivedRecordingReportCache.java

```diff
@@ -71,22 +71,24 @@ class ArchivedRecordingReportCache {
         this.logger = logger;
     }
 
-    Future<Path> get(String recordingName) {
+    Future<Path> get(String recordingName, String filter) {
         CompletableFuture<Path> f = new CompletableFuture<>();
+        System.out.println("I AM IN ARCHIVEDRECORDINGREPORTCACHE");
         Path dest = recordingArchiveHelper.getCachedReportPath(recordingName);
         if (fs.isReadable(dest) && fs.isRegularFile(dest)) {
+            System.out.println("IM HERE"); // cached
             f.complete(dest);
             return f;
         }
 
         try {
             logger.trace("Archived report cache miss for {}", recordingName);
-
+            System.out.println("ACTUALLY HERE"); // non-cached
```

Comment:
All these `printlns` will have to be removed before merge, but they're OK to keep for now while you're working through the implementation.